iMichaelOwolabi/store-manager
# #161124158 Store Attendant/Admin should be able to login

#### What does this PR do?
To review the changes made to the login page and the footer across pages
#### Description of Task to be completed?
To get this branch merged ultimately to the develop branch after acceptance
#### How should this be manually tested?
This can be tested out by visiting https://imichaelowolabi.github.io/store-manager/UI/ to see the current state of the page and refer to the attached screenshot to view the new changes.
#### Any background context you want to provide?
The redesign was needed because the page elements were not well placed in the previous design. The elements have been re-arranged and color scheme improved to be visually appealing
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2203575
#### Screenshots (if appropriate)
![login](https://user-images.githubusercontent.com/20207295/48621043-947d1800-e9a2-11e8-8459-133b51b814f7.PNG)

#### Questions:
@bobjayafam
Are there things I should change/edit or is this Ok to be merged as is? Thanks
